src/openrct2-ui/windows/Staff.cpp

```diff
@@ -887,9 +887,11 @@ void window_staff_options_invalidate(rct_window* w)
             w->pressed_widgets &= ~((1 << WIDX_CHECKBOX_1) | (1 << WIDX_CHECKBOX_2));
             w->pressed_widgets |= peep->StaffOrders << WIDX_CHECKBOX_1;
             break;
-        case STAFF_TYPE_SECURITY:
+        case StaffType::Security:
             // Security guards don't have an options screen.
             break;
+        case StaffType::Count: // All enumeration values must be added to prevent compilation error
```

Comment:
```suggestion
        case StaffType::Count:
```
No need for this comment